app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
+      },
+      {
+        role: 'redo'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'cut'
+      },
+      {
+        role: 'copy'
+      },
+      {
+        role: 'paste'
+      },
+      {
+        role: 'delete'
+      },
+      {
+        role: 'selectall'
+      }
+    ]
+  },
+  {
+    label: 'Help',
+    submenu: [
+      {
+        label: 'FAQ',
+        click () { require('electron').shell.openExternal('https://lbry.io/faq') }
```

Comment:
We have a help option in the UI. If we add help to the toolbar, it should go to the same place. If we want to encourage FAQ usage, the main "Help" section should do that.